lib/features/family/features/reflect/domain/models/roles.dart

```diff
@@ -38,7 +38,35 @@ class Reporter extends Role {
 
 class Sidekick extends Role {
   const Sidekick({
-    super.color = FamilyAppTheme.tertiary80,
+    super.color = ColorCombo.tertiary,
     super.name = 'sidekick',
   });
 }
+
+enum ColorCombo {
```

Comment:
Are these combos different from the ones we already had in the app? shared/models/colorcombo?

Reply:
yes, i tried to reuse but that one is big and getting confusing

Reply:
I could change the name tho